var/spack/repos/builtin/packages/binder/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Binder(CMakePackage):
+    """
+    Binder is a tool for automatic generation of Python bindings
+    for C++11 projects using Pybind11 and Clang LibTooling libraries.
+    That is, Binder, takes a C++ project and compiles it into objects
+    and functions that are all usable within Python.
+    Binder is different from prior tools in that it handles special
+    features new in C++11.
+    """
+
+    homepage = "https://github.com/RosettaCommons/binder"
+    git = "https://github.com/RosettaCommons/binder.git"
+
+    maintainers("lyskov", "kliegeois")
```

Comment:
@lyskov Are you okay with being a maintainer on this package?  (See https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers.)

Reply:
@tldahlgren this is a question about maintainer for spack package? If so then no, i do not think i should be listed as maintainer for it. Thanks,